agent/credentials/instancecreds/instancecreds_test.go

```diff
@@ -25,11 +25,10 @@ import (
 
 func TestGetCredentials(t *testing.T) {
 	credentialChain = nil
-	credsA := GetCredentials()
+	credsA := GetCredentials(false)
 	require.NotNil(t, credsA)
-	credsB := GetCredentials()
+	credsB := GetCredentials(true)
```

Comment:
Should this be `true` or should we have a specific external test case?

Reply:
We could have a specific test case to cover the new change but this should suffice too.